README.md

```diff
@@ -68,13 +68,14 @@ By environment variables:
 * [`kourou index:restore PATH`](#kourou-indexrestore-path)
 * [`kourou instance:logs`](#kourou-instancelogs)
 * [`kourou instance:spawn`](#kourou-instancespawn)
+* [`kourou query CONTROLLER:ACTION`](#kourou-query-controlleraction)
 * [`kourou vault:add SECRETS-FILE KEY VALUE`](#kourou-vaultadd-secrets-file-key-value)
 * [`kourou vault:encrypt FILE`](#kourou-vaultencrypt-file)
 * [`kourou vault:show SECRETS-FILE KEY`](#kourou-vaultshow-secrets-file-key)
 
 ## `kourou api-key:create USER`
 
-Creates a new API Key for a user.
+Creates a new API Key for an user
```

Comment:
https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

```suggestion
Creates a new API Key for a user
```

Reply:
I had to change it in the command description since the README is autogenerated